.github/CODEOWNERS

```diff
@@ -146,36 +146,46 @@
 # Dev workflow team.
 /.circleci/ @BenHenning @U8NWXD @kevintab95 @vinitamurthi
 # TODO(#10619): Add @bansalnitish as owner after 2020-09-26
-/.eslintignore @Hudda
+# TODO(#10709): Add @Hudda as owner after 2020-09-26
```

Comment:
We should find a way to avoid so many TODO comments, I'll add steps for doing this on wiki page, and from the future we will follow that pattern!

[Feel free to merge this PR!]